hnn_core/network_builder.py

```diff
@@ -318,6 +323,7 @@ def _build(self):
 
         self.move_cells_to_pos()  # position cells in 2D grid
         self._connect_celltypes()
+        self._record_lfp()
```

Comment:
move it next to `_record_spikes` ?

Reply:
It's actually really important this is placed after `self.move_cells_to_pos()`. When `LFPElectrode` is instantiated, it keeps track of where each cell is positioned to perform the voltage calculation. 

Do you think it's worth adding a comment to make sure people don't move it in the future?

Reply:
yes okay. Best would have been to have a check that if `pos_dict` is not created, then fail or something of that sort

Reply:
I guess the problem is `pos_dict` will always exist once this point is reached (since it's made in the Network instantiation), and the cells will have a default position when they are created. 

Maybe one way around this would be to pass the cell position to `LFPElectrode` through `net.pos_dict`, but I'm not sure if there's a huge benefit.

Reply:
I guess what I'm arguing for is a PR to remove `network_builder.move_cells_to_pos`. The transformation should happen before you go into Neuron. Ideally [after this line](https://github.com/jonescompneurolab/hnn-core/blob/master/hnn_core/network.py#L240)

Reply:
Good suggestion, it's honestly not clear to me why this doesn't just take place during `network_builder._create_cells_and_drives()`.